nixos/modules/services/desktops/playerctld.nix

```diff
@@ -0,0 +1,36 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.playerctld;
+in
+{
+  options.services.playerctld = {
+
+    enable = mkEnableOption (lib.mdDoc "Enable the playerctld daemon.");
+
+    package = mkPackageOption pkgs "playerctl" {
+      default = [ "playerctl" ];
```

Comment:
My reading of the `mkPackageOption` source code says that this should already be the default if it has the same name as above.

https://github.com/NixOS/nixpkgs/blob/ad7a8e6c27befbef28751586567027bedbcd72dc/lib/options.nix#L135

Reply:
corrected